at_client/src/main/java/org/atsign/client/util/ConvertingUtils.java

```diff
@@ -0,0 +1,60 @@
+import java.io.*;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+// Main class
+
+public class ConvertingUtils {
+
+     // Converting file object to byte array
+    public static byte[] convert(File file)
+        throws IOException
+    {
+  
+        //creating an instance of file
+        FileInputStream filel = new FileInputStream(file);
+  
+        //creating a byte array with same length as file
+        byte[] byteArr = new byte[(int)file.length()];
+  
+       //Reading file content 
+        filel.read(byteArr);
+  
+        //Closing the file
+        filel.close();
+  
+        // Returning byte array
+        return byteArr;
+    }
+
+  
+  //Converting byte array to file
+    public static File convert(byte[] bytes)
+    {
+        //paste yout destination path here
+        String FILEPATH="";  
+        File file = new File(FILEPATH);
+       
+        try {
+ 
+            // InitializING a pointer in file
+            OutputStream ostream = new FileOutputStream(file);
+            
+            // Starting writing the bytes in it
+            ostream.write(bytes);
```

Comment:
Does the file **have** to be written here? Or is it possible to just return the File object without writing it?